samples/tracing/README.md

```diff
@@ -0,0 +1,20 @@
+# Configuring tracing with eventing-rabbitmq
+
+For an overview of tracing with Knative in general, [this blog
+post](https://knative.dev/blog/articles/distributed-tracing/) introduces the
```

Comment:
Since I am reviewing this PR, I'm keeping post-related issues here:

```
kubectl create namespace observability &&
kubectl create -n observability \
    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml \
    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml \
    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml \
    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml \
    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
namespace/observability created
unable to read URL "https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml", server reported 404 Not Found, status code=404
unable to read URL "https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml", server reported 404 Not Found, status code=404
unable to read URL "https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml", server reported 404 Not Found, status code=404
unable to read URL "https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml", server reported 404 Not Found, status code=404
unable to read URL "https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml", server reported 404 Not Found, status code=404
```

Using the last version - [v1.28.0](https://github.com/jaegertracing/jaeger-operator/releases/tag/v1.28.0) - fixes it:

```diff
kubectl create -n observability \
-    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml \
+    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.28.0/deploy/crds/jaegertracing.io_jaegers_crd.yaml \
-    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml \
+    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.28.0/deploy/service_account.yaml \
-    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml \
+    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.28.0/deploy/role.yaml \
-    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml \
+    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.28.0/deploy/role_binding.yaml \
-    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
+    -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.28.0/deploy/operator.yaml
customresourcedefinition.apiextensions.k8s.io/jaegers.jaegertracing.io created
serviceaccount/jaeger-operator created
role.rbac.authorization.k8s.io/jaeger-operator created
rolebinding.rbac.authorization.k8s.io/jaeger-operator created
deployment.apps/jaeger-operator created
```

I think that we should do the same with CertManager. FWIW: https://github.com/knative-sandbox/eventing-rabbitmq/commit/a40b28453c8ac59d58cfdabf4e3a7bc48b4dc81f

Following the rest of the blog post.

Reply:
The `OpenTelemetryCollector` is failing to start:

```sh
kubectl apply -f - <<EOF
apiVersion: opentelemetry.io/v1alpha1
kind: OpenTelemetryCollector
metadata:
  name: otel
  namespace: observability
spec:
  config: |
    receivers:
      zipkin:
    exporters:
      logging:
      jaeger:
        endpoint: "simplest-collector.observability:14250"
        insecure: true

    service:
      pipelines:
        traces:
          receivers: [zipkin]
          processors: []
          exporters: [logging, jaeger]
EOF

kubectl logs deploy/otel-collector -n observability
Error: cannot unmarshal the configuration: error reading exporters configuration for "jaeger": 1 error(s) decoding:

* '' has invalid keys: insecure
2021/12/06 17:04:43 collector server run finished with error: cannot unmarshal the configuration: error reading exporters configuration for "jaeger": 1 error(s) decoding:

* '' has invalid keys: insecure
```

Reply:
Yeah, I mentioned in an earlier comment that I updated the tracing blog post to reflect these changes and lock the versions so it doesn't continue to drift: https://github.com/knative/docs/pull/4523

Reply:
Yes, I remember that.
I was expecting the change to be in already, but I can see that it was merged a few hours after I tried it out.
Re-running the steps from the blog now, which I expect to just work and be a noop.

Reply:
I'm not sure if this matters, but I've noticed that cert-manager is still pointing to the `latest` version.
While **I think** that [the current latest v1.6.1](https://github.com/jetstack/cert-manager/releases/tag/v1.6.1) just works, we may want to lock this one down as well.

I am currently running [cert-manager `v1.5.4`](https://github.com/jetstack/cert-manager/releases/tag/v1.5.4) in #525 since that's what [RabbitMQ Messaging Topology Operator v1.2.1](https://github.com/rabbitmq/messaging-topology-operator/releases/tag/v1.2.1) was tested against.

Reply:
cert-manager is pretty stable in my experience